db/migrate/20160412085547_create_custom_attribute_to_tag_mappings.rb

```diff
@@ -0,0 +1,13 @@
+class CreateCustomAttributeToTagMappings < ActiveRecord::Migration[5.0]
+  def change
+    create_table :custom_attribute_to_tag_mappings do |t|
+      t.string :resource_type
+      t.string :name
+      t.text :value
+      t.bigint :category_tag_id
+      t.bigint :tag_id
```

Comment:
* I like to use
```ruby
t.belongs_to :tag, :type => :bigint
```

It has no difference but its clearer.

* I think you can drop the "category_tag_id", i assume you can get it from the tag and place a ```:through``` ?